operate/client/src/App/ProcessInstance/TopPanel/MetadataPopover/v2/helpers.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import type {ElementType} from 'bpmn-js/lib/NavigatedViewer';
+import type {ElementInstance} from '@vzeta/camunda-api-zod-schemas';
+
+const formatElementTypeName = (
```

Comment:
I think this might be a bit more clear. What do you think?

```suggestion
const convertBpmnJsTypeToAPIType = (
```

Reply:
Good idea, renamed